cumulusci/tasks/apex/tests/test_apex_tasks.py

```diff
@@ -85,7 +86,10 @@ def _mock_apex_class_query(self, name="TestClass_TEST", namespace=None):
             "totalSize": 1,
         }
         responses.add(
-            responses.GET, url, match_querystring=True, json=expected_response
+            responses.GET,
+            url,
+            match=[query_string_matcher(query_string)],
```

Comment:
* [Deprecate](https://github.com/getsentry/responses/blob/071306524af1b4ee1797c45a46525274d271a9b6/CHANGES#L105) `match_querystring` argument in `Response` and `CallbackResponse`.
  Use `responses.matchers.query_param_matcher` or `responses.matchers.query_string_matcher`